docs/guides/mParticle-migration-guide.md

```diff
@@ -177,14 +182,18 @@ Both mParticle and Amplitude SDKs are meant to capture first party data by track
     <td>
     </td>
     <td>
+    Assign user to a group:
     ```swift
     Amplitude.instance().setGroup("orgName", groupName:NSString(string:"xyz"))
```

Comment:
Why the NSString here? Can we just use `Amplitude.instance().setGroup("orgName", groupName:"xyz")`

Reply:
I copied from these example in the iOS documentation
https://www.docs.developers.amplitude.com/data/sdks/ios/#swift_15
https://www.docs.developers.amplitude.com/data/sdks/ios/#swift_13